### DISCUSSION TITLE
What is the global ID used after enabling link tracing via enable_otlp_tracing?

### DISCUSSION THREAD

**Discussion by 1473371932** [GreptimeTeam/greptimedb#4563] [Q&A]
### What type of bug is this?

Configuration

### What subsystems are affected?

Standalone mode

### Minimal reproduce step

For example, after `jaeger` or `skywalking` is used for instrumentation, their default global ID is `uber-trace-id` or `sw8`. After greptimeDB starts call chain tracing, what is the global ID passed? It seems that there is no mention of this in the official documentation.

### What did you expect to see?

How can I get the global ID passed inside greptimeDB?

### What did you see instead?

Please ignore this

### What operating system did you use?

ubuntu 22.04

### What version of GreptimeDB did you use?

0.9.1

**Comment by evenyag** [comment]
https://opentelemetry.io/docs/concepts/context-propagation/#propagation
We use the default propagators. So it should be the [W3C TraceContext](https://www.w3.org/TR/trace-context/). The specification should mention the ID. I guess it is `traceparent`.

https://github.com/GreptimeTeam/greptimedb/blob/0f850370242ecb26261c5d43d4e2e4f0647729db/src/common/telemetry/src/logging.rs#L229

Converting this to discussion.
